---
layout: page
header: Enzyme Promiscuity
title: Structural analyses of generalist, specialist enzymes
description: Deciphering structual basis of enzyme promiscuity
img: assets/img/projects/enzyme_pros.jpg
importance: 1
category: work
---
<div class="row text-justify">
<b> Substrate promiscuity </b>
Enzyme promiscuity is the ability of (some) enzymes to catalyze alternate substrate(s) other than its native one. It is widely studied for its biotechnological applications and understanding enzyme evolution. Insights into the structural basis of substrate promiscuity would greatly benefit the design and engineering of enzymes. Previous studies on some enzymes have suggested that flexibility, hydrophobicity, and active site protonation state could play an important role in enzyme promiscuity. However, it is not known yet whether substrate promiscuous enzymes have distinctive structural characteristics compared to specialist enzymes, which are specific for a substrate. In pursuit to address this, we have systematically compared active site structural features of promiscuous with those of specialist enzymes and found them to be quite similar. We also proposed that substrate promiscuity could be defined as a continuum feature that varies from narrow (specialist) to broad range of substrate preferences. 

Additionally,  we are interested in following-
<ul>
<li> If structure doesn't then what dictates the segregation between promiscuous and specialist enzymes ? </li>
<li> Is there a presence of allosteric regulation via hidden cryptic binding sites or change in interaction partners in different organisms or both ? </li>
<li> What is the effect of promicuous activities on the metabolic pathways?</li>
</ul>

Information and observations gained from above analysis will help not only knowing more details about enzymes but also enhance their commercial application, which if can be tuned allosterically by mimicking what triggers promiscuity/specialty may provide tremendous biotechnological wealth. Moreover, this also holds promising value in modeling gut microbiome and metabolite interactions, where detailed impacts of such promiscuous activities have yet to be studied.
<br>


<div class="row align-center">
    <div class="col mt-3 mt-md-0">
        {% include figure.html path="assets/img/prom.gif" title="example image" class="img-fluid rounded z-depth-1" %}
    </div>
</div>

<div class="caption">
Image showing docked alternate substrates
</div>

<div class="row">

<div class="text-capitalize font-weight-bold"> Flux Balance analysis</div>
Genome Scale Models are metabolic network reconstructions of a target organism, built using all the existing knowledge about that organism, assembled in the form of a stoichiometric matrix on which various mathematical operations are performed to predict the metabolic phenotypes of the target organism under different genotypic or environmental perturbations. Previous studies have established that metabolic pathways are robust under various perturbations and in many instances, the rewired pathways involves promiscuous enzymes. It has speculated that enzyme evolve from them under stress conditions. We are use FBA approach to investigate the growth of gene deletion strains of E. coli under various nutrient conditions for identifying alternate pathways, which are present in these growth conditions. 

</div>
</div>
